sql/dd/impl/types/schema_impl.cc

```diff
@@ -106,6 +106,25 @@ bool Schema_impl::set_options_raw(const String_type &options_raw) {
   return false;
 }
 
+bool Schema_impl::set_db_metadata(const String_type &metadata) {
+  if (metadata.empty()) {
+    if (options().exists("db_metadata")) {
```

Comment:
Can "db_metadata" be made into a constant string, similar to `read_only_options_key`?